docs/testing.rst

```diff
@@ -265,3 +265,91 @@ does hurt to double check though.
 
 .. _Betamax: https://github.com/sigmavirus24/betamax
 .. _cassettes: https://betamax.readthedocs.org/en/latest/cassettes.html
+
+Asserting using the assert statement
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``py.test`` allows you to use the standard python ``assert`` for verifying expectations and values in Python tests. For example, you can write the following:
+
+.. code::
+
+    # content of test_assert1.py
+    def f():
+        return 3
+
+    def test_function():
+        assert f() == 4
+        
+to assert that the  function returns a certain value. If the assertion fails you will see the return value of the function call:
+
+.. code::
+
+     $ py.test test_assert1.py
+    ======= test session starts ========
+    platform linux -- Python 3.4.3, pytest-2.8.2, py-1.4.30, pluggy-0.3.1
+    rootdir: $REGENDOC_TMPDIR, inifile:
+    collected 1 items
+
+    test_assert1.py F
+
+    ======= FAILURES ========
+    _______ test_function ________
+
+        def test_function():
+    >       assert f() == 4
+    E       assert 3 == 4
+    E        +  where 3 = f()
+
+    test_assert1.py:5: AssertionError
+    ======= 1 failed in 0.12 seconds ========
+    
+
+Making use of context-sensitive comparisons
```

Comment:
Now it seems like you've just copied content from pytest's documentation. Everything from here to the end of the file.